docs/pages/admin-guides/access-controls/access-graph/ssh-keys-scan.mdx

```diff
@@ -0,0 +1,204 @@
+
+## How it works
+
+Teleport and Teleport Policy's Access Graph synchronize various resources, including SSH authorized keys and private keys.
+
+These resources are then visualized using the graph representation detailed in the
+[Access Graph page](../teleport-policy/getting-started-policy.mdx).
+
+### Importing Node's SSH Authorized Keys
+
+Teleport Cluster Nodes running the Teleport SSH Service continuously scan for SSH Authorized Keys present on the server.
+The public fingerprint of these keys is sent to the Teleport Auth Service, which then pushes them to the Access Graph.
+Together with the key fingerprint, the node also sends the following metadata:
+
+- `public_fingerprint`: The SHA256 fingerprint of the public key.
+- `host_id`: The unique identifier of the node.
+- `username`: The local username whose access is granted by the key.
+- `key_comment`: The comment associated with the key in the `authorized_keys` file.
+
+By default, Teleport scans the `$HOME/.ssh/authorized_keys` and `$HOME/.ssh/authorized_keys2` files for authorized keys.
+
+### Importing User's SSH Private Keys
+
+Teleport's `tsh` CLI tool can scan users' laptops for SSH private keys.
+It goes through the specified directories—defaulting to `/Users` on macOS, `/home` on Linux, and `C://Users` on Windows—
+by examining files with certain prefaces to identify SSH private keys.
+
+The `tsh` tool authenticates with the Teleport Cluster through the [Device Trust](../device-trust.mdx) feature,
+which guarantees that only enrolled devices can submit private keys to the cluster. By utilizing the device's Secure
+Enclave or TPM private key, it confirms that the device is the same one that was enrolled, enabling Teleport to trust
+and accept the private key reports without requiring further authentication or credentials thus allowing scanning operations
+to be performed without user interaction and valid credentials.
+
+Once the device is authenticated, the `tsh` tool extracts the public key fingerprint from the private key, if available,
+otherwise, it generates the fingerprint from the public key file if the last exists. The fingerprint is then sent to the
+Teleport Cluster with metadata such as:
+
+- `device_id`: The unique identifier of the device.
+- `public_fingerprint`: The SHA256 fingerprint of the public key if available.
+- `derivation_mode`: The method used to derive the public key from the private key. If can be either `private_key` if the
+  public key is extracted from the private key, `public_key_file` if the private key is encrypted and the public key is
+    available in a separate file, or `protected` if the public key fingerprint is not available.
+
+<Notice type="warning">
+`tsh` never sends the private key itself to the Teleport Cluster, ensuring that the private key remains secure on the device.
+It also never sends the private key path or any other sensitive information.
+</Notice>
+
+### Creating insecure access paths
```

Comment:
I don't know if this section is necessary. It seems to just reiterate topics that have already been discussed.

Reply:
removed